holidays/financial/australian_securities_exchange.py

```
@@ -0,0 +1,185 @@
+        self._add_holiday_1st_mon_of_oct(
+            # Labour Day.
+            self._format_holiday_name(no_settlement_label, tr("Labour Day"))
+        )
+
+        if self._year >= 2009:
+            self._add_holiday_1st_tue_of_nov(
+                # Melbourne Cup Day.
+                self._format_holiday_name(no_settlement_label, tr("Melbourne Cup Day"))
+            )
+
+    def _populate_half_day_holidays(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_populate_half_day_holidays`

Add return type annotation: `None`

(ANN202)